marketdata/stream/message_test.go

```diff
@@ -179,6 +181,7 @@ var testTradingStatus = tradingStatusWithT{
 	Type:       "s",
 	Symbol:     "BIIB",
 	StatusCode: "T",
+	StatusMsg:  "Trading Resumption",
```

Comment:
Please add `ReasonMsg` as well (and change `ReasonCode` to something that's not empty)